.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     parallelism: 1
     docker:
-      - image: circleci/elixir:1.12.1
+      - image: cimg/elixir:1.13
```

Comment:
I think we should match `.tool-versions` with 1.13. E.g.
```
elixir 1.13.0-otp-24
erlang 24.1.7
```
How that sounds?

Reply:
👍